Automattic/wp-calypso
# Add Related Plugins component

Closes #80288

## Proposed Changes

* add the related plugins component

## Testing Instructions


* Visit `/plugins`
* Click on any plugin, and scroll down to the bottom of the details page
* Check the component design against the Figma design here: f84fqtCG4pCnCh2Hq0But5-fi-2011_24544
* Check that pricing is displayed correctly for both free and paid plugin

<img width="640" alt="CleanShot 2023-08-28 at 15 52 46@2x" src="https://github.com/Automattic/wp-calypso/assets/11555574/6410d24a-7083-48fc-bff8-321fc24ed09c">

Test the `See All` link
- Edit the [`client/my-sites/plugins/plugin-details.jsx`](https://github.com/Automattic/wp-calypso/pull/81081/files#diff-88b330820980195a5fac3dab16832001fa668d33d4ae537321f4e16794dbb33fR368) file
- Add this to line `368`: `seeAllLink="/plugins"`
- Check that the link is displayed and that it redirect to the plugins page

<img width="608" alt="CleanShot 2023-08-29 at 13 30 47@2x" src="https://github.com/Automattic/wp-calypso/assets/11555574/7a667e2c-e4f7-4c21-935e-01c57b8bf6e1">

## Pre-merge Checklist
